index.html

```diff
@@ -4,9 +4,126 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <title>RoofTop-Cafe</title>
+    <link rel="stylesheet" href="styles.css">
+    <link href='https://fonts.googleapis.com/css?family=Lato' rel='stylesheet'>
 </head>
 <body>
-    
+    <header>
+    <div class="nav-menu">
+        <i><img src="./images/menu-bar.jpg" alt="menu bars"></i>
+    </div>
+    <div class="mobile-menu">
+        <ul>
+            <li><a href="./index.html">Home</a></li>
+            <hr class="line-mobile-menu">
+            <li><a href="./about.html">About</a></li>
+            <hr class="line-mobile-menu">
+            <li><a href="#">Menu</a></li>
+            <hr class="line-mobile-menu">
+            <div class="closeMenu"><i><img src="images/cross-sign.png" alt="cross icone"></i></div>
+        </ul>
+    </div>
+
+    <div class="nav-black">
+        <div class="nav-icons">
+        <ion-icon name="logo-facebook" class="icon"></ion-icon>
+        <ion-icon name="logo-twitter" class="icon"></ion-icon>
+        <span class="tabs">English</span>
+        <span class="tabs">Language</span>
+    </div>
+      </div>
+
+        <div class="nav-desktop">
+            <div class="nav-logo">
+                <a href="./index.html"><img src="./images/logo.png" alt="Logo"></a>
+            </div>
+            <div class="nav-items">
+                <ul>
+                    <li><a href="./about.html">About</a></li>
+                    <li><a href="#">Contact</a></li>
+                    <li><a href="#">Menu</a></li>
+                </ul>
+            </div>
+        </div>
+    </header>
+    <main>
+    <section id="header">
+        <div class="subheading"><h2>"A taste you will remember"</h2>
+        </div>
+    <div class="title"><h1>Roof Top-Cafe</h1></div>
+    <div class="details"><p>This is the place to have delicious food which always comes up with fresh food for its customers. Because their satisfactions and gratifications about food freshness and services is our basic axiom and responsibility.</p></div>
+    <div class="address">
+        <h4>@Main Road Vanike Tararr, Hafizabad</h4>
+    </div>
+</section>
+<section id="dishes">
+    <div class="dishes-heading"><h2>Main Dishes</h2></div>
+    <div class="orangeline"></div>
+    <div class="dish-details">
+    <div class="dish"><img src="./images/grill-fish.png" alt="Grill Fish">
+    <h3>Grill Fish</h3>
```

Comment:
- [ ] As requested by the previous reviewer, Please add some padding to the cards both top and bottom to make them match the original design. This will make the card look much better from a UI perspective.


| ------------- | ------------- |
| ------------- | ------------- |
| Your Design  | <img width="825" alt="image" src="https://user-images.githubusercontent.com/52934128/217894532-ef773dbf-7c7b-4300-9de1-fa4b43362eff.png">  |
| Template Design  | <img width="804" alt="image" src="https://user-images.githubusercontent.com/52934128/217893963-b166e9d8-fcd6-400d-b52c-84c23a5c7da4.png">|